loadouts/factions/Korea.sqf

```diff
@@ -0,0 +1,261 @@
+// INFO ========================================================================
+/*
+   South korean army loadout.
+*/
+
+_factionID = "ROKA";
+_factionName = "ROK Army";
+_factionStructure = "HIGH-TIER-CCO";
+
+_factionCallsigns = _defaultCallsignBLUFOR;
+
+_defaultSide = WEST;
+_defaultVoice = [_voiceAMERICAN];
+_defaultFace = [_faceASIAN];
+_defaultName = [_nameAMERICAN];
+_defaultInsignia = "USP_PATCH_ROK_ARMY_9INF_DIV";
+_defaultColor = "blue";
+
+/*              "Accuracy", "Aiming Shake", "Aiming Speed", "Endurance", "Spoting Distance", "Spotting Time", "Courage", "Reloading Speed", "Commanding", "General" */
+_factionSkill = [[0.7,0.8],   [0.8,0.9],      [0.7,0.8],     [0.7,0.9],      [0.8,0.9],        [0.7,0.8],     [0.8,0.9],     [0.7,0.8],      [0.7,0.9],   [0.7,0.8]];
+
+// CAMO ========================================================================
+/*
+    Set in mission with: BRM_FMK_LoadoutCamo_ROKA
+	  "GRANITE"
+    "ROKMC"
+    "DESERT"
+*/
+_camo = "GRANITE";
+if(!isNil"BRM_FMK_LoadoutCamo_ROKA")then{_camo=BRM_FMK_LoadoutCamo_ROKA};
+_lp = "GraniteB"; // Loadout Pattern
+_rhp = "fg"; // Recon Head Pattern
+
+if (_camo == "ROKMC") then {
+	_lp = "ROKMC";
+    _rhp = "fg";
+};
+
+// WEAPONS =====================================================================
+
+_commonRIFLE = ["BRM_K2","rhs_mag_30Rnd_556x45_M855_Stanag"];
+_reconRIFLE = ["BRM_K2C1","rhs_mag_30Rnd_556x45_M855_Stanag"];
+_commonRIFLEGL = ["BRM_K2GL","rhs_mag_30Rnd_556x45_M855_Stanag","1Rnd_HE_Grenade_shell"];
+_commonPISTOL = _P228;
+_commonAR = _FNMINIMI;
+_commonMG = ["R3F_Minimi_762_PARA","R3F_MINIMI_100Rnd_762x51"];
+_commonMARKSMAN = _DMRRIS;
+_commonSNIPER = _AWM;
+_commonAT = _AT4;
+_specAT = _GUSTAV;
+_weaponsAA = _STINGER;
+_weaponsAT = _JAVELIN;
+_commonSMG = _RHSM4;
+_commonRCO = "BRM_K2CS";
+_marksmanSCOPE = "rhsusf_acc_m8541_mrds";
+_specATSCOPE = "rhs_optic_maaws";
+_commonCCO = "BRM_K2CQS";
+_commonEOT = "rhsusf_acc_eotech_552";
+_commonRAIL = "rhsusf_acc_anpeq15side_bk";
+_commonBIPOD = "rhsusf_acc_tdstubby_blk";
```

Comment:
Is incompatible with all of the weapons:
	`_commonRIFLE`
	`_reconRIFLE`
	`_commonRIFLEGL`
	`_commonAR`
	`_commonMARKSMAN`
	`_commonMG`